columnphysics/icepack_parameters.F90

```diff
@@ -250,7 +250,7 @@ module icepack_parameters
          TTTocn = 5107.4_dbl_kind      ! for qsat over ocn
 
       character (len=char_len), public :: &
-         atmbndy = 'default' ! atmo boundary method, 'default' ('ccsm3') or 'constant'
+         atmbndy = 'similarity'        ! atmo boundary method, 'similarity' ('ccsm3'), 'constant' or 'mixed'
```

Comment:
I wonder if we should use this opportunity ("while we're here" kind of thing) to remove the `('ccsm3') `in the comment (and do the same in `icepack_init_parameters`.)  It has been there ever since the first commit (6f98a36) but 'ccsm3' was never a value for 'atmbndy', at least not in Icepack.


In fact it dates back to this commit: https://github.com/CICE-Consortium/CICE-svn-trunk/commit/dfa799dfb8a0eec78e4a976c82c25b869f3e959e and so it was never an acceptable value at all!